### ISSUE TITLE
Sankey | Nodes not focusable with keyboard navigation

### ISSUE THREAD

**Issue by acostales19** [highcharts/highcharts#18497]
#### Expected behaviour
To be able to use arrow key to focus on links AND nodes (all points) on Sankey diagram.

#### Actual behaviour
Arrow keys only set focus on links.

#### Live demo with steps to reproduce
[Sankey demo](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/demo/sankey-diagram)

#### Product version
Highcharts 9.3.3 

#### Affected browser(s)
Edge

**Comment by oysteinmoseng** [comment]
Internal note: There is unfortunately no quick workaround for this - Sankey series use `series.nodes` for the nodes instead of adding them to `series.points`. 

We should fix this as part of the advanced keyboard navigation work, as a tree-based navigation model would perhaps be more suited for this kind of chart (at least it should be an option).
